go.mod

```diff
@@ -5,14 +5,14 @@ go 1.14
 replace k8s.io/client-go => github.com/rancher/client-go v0.18.8-fleet1
 
 require (
-	github.com/rancher/eks-operator v0.1.0-rc29
+	github.com/rancher/eks-operator v1.0.6-rc1
```

Comment:
We'll have to remember to bump this to non-rc version once its tagged. Can you track that as either an issue or a personal task.

Reply:
Yes, it will have to be updated in rancher, system-charts, and here.